src/main/commandServer/settingsValidator.ts

```diff
@@ -229,6 +235,8 @@ export default class SettingsValidator {
     return currentValue !== desiredEngine;
   }
 
+  protected checkMutedDiagnostics = this.checkWSLIntegrations;
```

Comment:
We may need to rename the method to make it clearer that it's shared… (so we don't end up changing the implementation and forget that it also applies to mute settings).

Reply:
We discussed offline and decided to rename `checkWSLIntegrations` to `checkBooleanMapping`